_docs/developer/vm_install_using_vagrant.md

```diff
@@ -398,15 +398,30 @@ instructions.
 
    Replace `<port>` with the port you want to expose externally on the machine that is running the VM, and expose the specified port on the machine if necessary.
 
-3. Start the vagrant machine with `vagrant up ssh`.
+3. Start the vagrant machine with the SSH argument set:
 
-4. Retrieve the private key for the vagrant machine, located at `<SUBMITTY GIT REPO>/.vagrant/machines/<VM>/VM BACKEND>/private_key`.
+   **Windows**
+   ```
+   set SSH=y
+   vagrant up
+   ```
+
+   **macOS/Linux**
+   ```
+   SSH=y vagrant up
+   ```
+
```

Comment:
I think the SSH argument isn't necessary anymore, at least with my testing on Vagrant 2.2.16 I just have to do vagrant up.